cmd/generic-handlers.go

```diff
@@ -154,15 +154,15 @@ func setRedirectHandler(h http.Handler) http.Handler {
 
 func guessIsBrowserReq(r *http.Request) bool {
 	aType := getRequestAuthType(r)
-	ok := strings.Contains(r.Header.Get("User-Agent"), "Mozilla") && globalBrowserEnabled &&
-		(aType == authTypeJWT || aType == authTypeAnonymous)
-	return ok
+	return strings.Contains(r.Header.Get("User-Agent"), "Mozilla") &&
+		globalBrowserEnabled && aType == authTypeAnonymous
```

Comment:
Maybe you need to remove this as well ?  aType == authTypeAnonymous

It is weird that guessIsBrowserReq doesn't return true when this is authenticated request from the browser

Reply:
There is no authentication coming to MinIO @vadmeste on browser anymore